faqs/zh/governance.md

```diff
@@ -215,16 +215,30 @@ MKR 持有者的任务是维护系统的健康以及 Dai 的稳定锚定，同
 
 ## 在连接后，如果我将更多 MKR 发送到冷钱包里会怎么样？
 
-你可以选择将新 MKR 添加到您的投票合约中进行充值，只需要一笔交易即可。可以通过打开 [治理面板](https://vote.makerdao.com)，点击“投票合约”，然后点击“充值投票合同”按钮找到此选项。
+你可以选择将新 MKR 添加到您的投票合约中进行充值，只需要一笔交易即可。可以通过打开 [治理面板][12]，点击“投票合约”，然后点击“充值投票合同”按钮找到此选项。
 
 
 ## 我可以断开我冷热钱包之间的连接吗？
 
-可以。您可以在 [治理面板](https://vote.makerdao.com) 上断开钱包间的连接。点击“投票合约”，然后点击“断开钱包连接”按钮。
+可以。您可以在 [治理面板][13] 上断开钱包间的连接。点击“投票合约”，然后点击“断开钱包连接”按钮。
 
 一般来说，除非你需要选择新的热钱包或冷钱包，没有理由断开连接。你可以在不断开连接的情况下，从投票合同中取回 MKR，自由地使用冷钱包中的 MKR。
 
 
 ## 我可以合并多个持有 MKR 的地址进行投票吗？
 
-不可以，目前投票合约只能由一个地址提供资金。
\ No newline at end of file
+不可以，目前投票合约只能由一个地址提供资金。
+
+[1]:	http://vote.makerdao.com/
+[2]:	https://mp.weixin.qq.com/s/ENiFzoQo3F79FVzgBFHIEQ
+[3]:	http://vote.makerdao.com/
```

Comment:
Couldn't all the links to http://vote.makerdao.com/ just point to `[1]`?

Do we need so many duplicates?

Reply:
I'm not sure about this syntax at all, it's first time I see it, but perhaps it could point to [1].

Reply:
Might be a different syntax with the editor. It does not duplicate in the display, but will modify to the standard.